recipes/flock/meta.yaml

```diff
@@ -0,0 +1,13 @@
+package:
+  name: flock
+  version: "1.0"
+source:
+  git_url: https://github.com/cristhomas/immport-test.git
```

Comment:
thank you, yes it does. 
how did you get it to work? I couldn't find any help online with the error message I was getting.